extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/HueErrorCodeDescriptor.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.smarthome.binding.hue;
+
+public enum HueErrorCodeDescriptor {
+	BRIDGE_NOT_RESPONDING(1, "The hue bridge doesn't respond. Please doublecheck the provided ip address."),
+    IP_ADDRESS_MISSING(2, "No ip address for the hue bridge has been provided.");
+	
+	public static final String CONFIG_FLOW_ERROR_PREFIX = "setup-flow.hue.error.";
+
+	private int code;
+    private String message;
+
+    private HueErrorCodeDescriptor(int code, String message) {
```

Comment:
I would prefer if we rename this enum to HueConfigStatusMessage... then you could define all the required attributes for the config status message